phpcs.xml.dist

```diff
@@ -19,6 +19,7 @@
         <exclude name="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint"/>
         <exclude name="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingReturnTypeHint"/>
         <exclude name="SlevomatCodingStandard.Exceptions.ReferenceThrowableOnly.ReferencedGeneralException"/>
+        <exclude name="SlevomatCodingStandard.ControlStructures.EarlyExit"/>
```

Comment:
I guess this still breaks too much stuff

Reply:
Most of the issues were fixed, but I didn't want to do this here as it would make the diff a lot harder to review, rather separately, some code could even be refactored using private methods.